docs/api.md

```diff
@@ -610,6 +612,26 @@ const browser = await puppeteer.launch({
 >
 > See [`this article`](https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/) for a description of the differences between Chromium and Chrome. [`This article`](https://chromium.googlesource.com/chromium/src/+/lkgr/docs/chromium_browser_vs_google_chrome.md) describes some differences for Linux users.
 
+#### puppeteer.networkConditions
```

Comment:
NIT: naming. We use `networkConditions` as a single instance and as a list of conditions. Can we rename this method to something like `networkConditionsList`?

Reply:
It‘s not a method, but rather an object. Given that this is analogous to [`puppeteer.devices`](https://github.com/puppeteer/puppeteer/blob/v5.5.0/docs/api.md#puppeteerdevices), I think we should stick to the current name.

Reply:
Sounds good to me!